yo/app/scripts/controllers/model.js

```diff
@@ -149,6 +149,24 @@ angular.module('dmpApp')
             PubSub.broadcast('restoreCurrentProject', {});
         };
 
+        function showChild(needle, child) {
+
+            if(child.id === needle) {
+
+                child.$wasRendered = true;
+                child.$show = true;
+
+                return;
+            }
+
+            if(child.hasChildren) {
+                loDash.map(child.children, function(child) {
```

Comment:
Not that important (i.e. no need for imminent change), but this could be forEach instead of map (here and the other usages).